ratbagd/ratbagd.c

```diff
@@ -39,6 +39,8 @@
 #include "shared-macro.h"
 #include "libratbag-util.h"
 
+#include "ratbagd-test.h"
```

Comment:
Why not do the `ifdef` here?

Reply:
we still need the `ratbagd_init_test_device` unconditionally (though that can probably be worked around as well. Plus, I'm not a big fan of ifdef'd includes